1-development/list_posts_by_rating/src/main/java/com/mixfar/posts/utils/Utils.java

```diff
@@ -0,0 +1,23 @@
+package com.mixfar.posts.utils;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+public class Utils {
+    public static long getDaysUntilNow(LocalDate date){
+        return date.until(LocalDate.now(), ChronoUnit.DAYS);
+    }
+
+    public static float calculateRating(Long upvotes, Long downvotes, long days){
```

Comment:
I like this approach. What are the ```advantages``` and ```disadvantages``` of this?

Reply:
By calculating rating with taking into account the age of the post we can do a more precise rating, the disadvantages of this are that when posts get older the rating starts to decrease